content/en/docs/concepts/certificaterequest.md

```diff
@@ -72,170 +97,498 @@ Each ready condition consists of the pair `Ready` - a boolean value, and
+      - "awspcaclusterissuers.awspca.cert-manager.io/awspca-issuer"
+```
+
+Note: the "resource type"
+[is](https://kubernetes.io/docs/reference/using-api/api-concepts/#standard-api-terminology)
+the lower-case and plural version of the "kind" field.
+
+
+### Syntax of the `resourceNames` field used in RBAC roles {#approval-api-rbac-syntax}
+
+When a user or controller attempts to approve or deny a CertificateRequest, the
+cert-manager webhook evaluates whether it has sufficient permissions to do so.
+These permissions are based upon the request itself, and more specifically, the
+CertificateRequest's `issuerRef` field.
+
+{{% pageinfo color="warning" %}}
+
+The verb `approve` on the resource `signer` only works with `ClusterRole`, and
+the `apiGroup` must be set to `cert-manager.io`.
+
+{{% /pageinfo %}}
+
+Let's imagine I want to use this approver with these two issuers:
+
+```yaml
+apiVersion: awspca.cert-manager.io/v1beta1
+kind: AWSPCAClusterIssuer
+metadata:
+  name: private-ca
+spec:
+  arn: arn-c1f4f80cd072
+  region: eu-west-1
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: letsencrypt
+  namespace: namespace-with-letsencrypt
+spec:
+  acme: {}
+```
+
+The RBAC roles to be bound to the service account of your approver is:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
 rules:
   - apiGroups: ["cert-manager.io"]
     resources: ["signers"]
     verbs: ["approve"]
-    resourceNames: ["myissuers.my-example.io/myapp"]
+    resourceNames:
+      # Allow this specific AWSPCAClusterIssuer issuer:
+      - "awspcaissuers.awspca.cert-manager.io/private-ca"
+      #  <-----------> <--------------------> <------------------> <-------->
+      #  resource type     resource group          signer name     issuer name
+
+      # Or if you prefer allowing any AWSPCAClusterIssuer:
+      - "awspcaissuers.awspca.cert-manager.io/*"
+
+
+      # For namespaced issuers, the syntax is similar:
+      - "issuers.cert-manager.io/namespace-with-letsencrypt.letsencrypt"
+      #  <-----> <-------------> <-------------------------> <--------->
+      #    type       group            issuer namespace      issuer name
+
+      # You can also allow any issuer type for namespaced issuers:
+      - "issuers.cert-manager.io/*"
+```
+
+If the approver does not have sufficient permissions to set the `Approved` or
+`Denied` conditions, the CertificateRequest is rejected by the cert-manager
+webhook.
+
+### Reason and message of the Approval API
+
+The `reason` and `message` fields on the `Denied` and `Approved` condition
+are approver-specific, meaning that each approver will set their own. Here
+is the set of guidelines:
+
+| Field     | Guideline                                               |
+| --------- | ------------------------------------------------------- |
+| `reason`  | Machine-readable string identifying the approver        |
+| `message` | Human-readable explanation of why the condition was set |
```

Comment:
This is moving away from the original intention. Please add back the original definitions.

```
The Reason field of the Approved/Denied condition should be set to *who* set the
condition. Who can be interpreted however makes sense to the approver
implementation. For example, it may include the API group of an approving policy
controller, or the client agent of a manual request.
````


````
The Message field of the Approved/Denied condition should be set to *why* the
condition is set. Again, why can be interpreted however makes sense to the
implementation of the approver. For example, the name of the resource that
approves this request, the violations which caused the request to be denied, or
the team to who manually approved the request.
```